domains/killerf.json

```
@@ -1,11 +1,11 @@
 {
-  "description": "The Person Who Hacked QLASH RP And got away with it with nothing to lose, An Exploiter to say the least; Welcome to Grove Street",
-  "repo": "https://discord.gg/groove-community-875725991551569960",
+  "description": "The Person Who Hacked QLASH RP And got away with it with nothing to lose, An Exploiter to say the least; Welcome to the other side",
+  "repo": "https://guns.lol/killerf",
   "owner": {
-    "username": "MohamexDR",
+    "username": "GS KillerF",
```

username needs to be your github username
```suggestion
    "username": "MohamexDR",
```